### PR TITLE
Extractosm be

### PR DESCRIPTION
**Pull request by wlangera**

### CODE REVIEW

File: src/R/steekproeftrekking_nabehandeling.R [hunk 1]
```diff
@@ -65,7 +65,7 @@ bereken_zichtbaarheid <- function(point,
   return(out)
 }
 
-filter_zichtbaarheid <- function(steekproef, min_cvvi) {
+filter_zichtbaarheid <- function(sample, min_cvvi) {
   steekproef %>%
```
Comment (hansvancalster) [inline src/R/steekproeftrekking_nabehandeling.R:69]:
```suggestion
  sample %>%
```

**Review by hansvancalster — APPROVED** [review]
Looks good! Just one thing that needs to be changed I guess

**Comment by wlangera** [comment]
@hansvancalster I accepted your change and also added an feature to update the osm layer if desired.

**Comment by hansvancalster** [comment]
> @hansvancalster I accepted your change and also added an feature to update the osm layer if desired.

Good idea 👍

**Pull request merged**